docs/source/install.md

```diff
@@ -129,6 +136,32 @@ The location of the GDAL DLLs must be on your system `PATH`.
 `--no-use-pep517` is required in order to pass additional options to the build
 backend (see https://github.com/pypa/pip/issues/5771).
 
+#### Conda
+
+It is also possible to install the necessary dependencies using conda.
+
+After cloning the environment, you can create a conda environment with the necessary
+dependencies like this:
+```
+conda env create -f environment-dev.yml
+```
+
+Before being able to build on Windows, you need to set some additional environment
+variables or because the location of the GDAL binaries and headers cannot be
+automatically determined.
+
+After activating the pyogrio-dev environment the CONDA_PREFIX environment variable will
+be set and you will be alble to set the necessary environment variables as follows:
+
+```bash
+set GDAL_INCLUDE_PATH=%CONDA_PREFIX%\Library\include
+set GDAL_LIBRARY_PATH=%CONDA_PREFIX%\Library\lib
+set GDAL_VERSION=3.7.0
```

Comment:
Is it possible to set this dynamically?  For our windows tests, we're deriving this dynamically `GDAL_VERSION=$(gdalinfo --version | cut -c 6-10)`

See `.github/workflows/tests-conda.yml`.

Would be good to not have this value hard-coded, if possible.

Reply:
No. That command only works in a bash shell window, which isn't very common on windows. In the standard shells, cmd or powershell, this doesn't work. The tooling available in e.g. cmd is also very limited, so not trivial (at least to me) to write something similar and even worse to have it work both in powershell and cmd.

I haven't looked into it and am not familiar with it, but isn't it possible to determine the version in the build script? If I recall correctly the GDAL_VERSION environment variable wasn't needed in the beginning.

Reply:
We use it early on to check that it is a supported version of GDAL.  Thanks for considering more general approaches; your updates to this section are fine.